src/libs/curated_app_libs.py

```diff
@@ -199,7 +197,6 @@ def run_test(test_instance, test_yaml_file):
         workload_name = get_workload_name(test_config_dict['docker_image'])
         curation_output = generate_curated_image(test_config_dict, run_with_test_option)
         if "expected_output_infile" in test_config_dict.keys():
-            cleanup_after_test(workload_name)
```

Comment:
Why was this removed?

Reply:
this will be taken care in finally statement...

Reply:
No, it will not be able to... because there is return before that